.github/workflows/reliably.yaml

```diff
@@ -9,8 +9,8 @@ jobs:
     steps:
       - name: 'Checkout source code'
         uses: actions/checkout@v2
-      - name: 'Run Reliably'
-        uses: reliablyhq/gh-action@main
+      - name: 'Run Reliably Scan'
```

Comment:
This Action needs to authenticate with Reliably via the `RELIABLY_TOKEN` and `RELIABLY_ORG` variables like in the example: https://github.com/reliablyhq/actions/tree/master/scan

You can see that the action step passes but actually it just tells you it needs to login:https://github.com/chaostoolkit/walkthrough/runs/3290459037?check_suite_focus=true#step:4:10

Reply:
Bloody Github, this isn't an approve

Reply:
To login, you need to do that from your local machine and use your token as a secret on this repo. I can help on this. Unfortunately, we do not support yet org level token so that'll be yours :p

Reply:
Like this? I'm rather lost

Reply:
Welp, no, you'll need to re-login now, that token is out in the open. 

I'll show you @CharlieMoon37